src/components/HomeSponsors.vue

```diff
@@ -24,14 +24,18 @@ import logoITK from '@/assets/sponsors/ITK.png'
 import logoAkio from '@/assets/sponsors/akio.png'
 import logoZendesk from '@/assets/sponsors/zendesk-medium.png'
 import logoAusy from '@/assets/sponsors/ausy.png'
+import logoTeads from '@/assets/sponsors/teads.tv_noir.png'
```

Comment:
you should rename Teads file to prevent ad blocker which will block all logos

Reply:
I always forgot about ad blocker :( 
Any idea for image naming ?

Reply:
:joy: "sdeat" :rofl:

Reply:
did `teadz` ok ?

Reply:
let's try :D